flake.nix

```diff
@@ -0,0 +1,53 @@
+{
+  description = "Flake for Plutus";
+
+  inputs = {
+    nixpkgs.url = "github:NixOS/nixpkgs?rev=7d71001b796340b219d1bfa8552c81995017544a";
+    haskell-nix.url = "github:input-output-hk/haskell.nix?rev=87084d65a476cc826a0e8c5d281d494254f5bc7a";
+    flake-utils.url = "github:numtide/flake-utils";
```

Comment:
pin?

Reply:
Yeah let me pin that as well. For some reason only `nixpkgs` was pinned in the PR branch that I started this off from..